mission/task_manager/scripts/task_manager_server_node.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/python3
+import rospy
+
+from dynamic_reconfigure.server import Server
+from task_manager.cfg import NjordTasksConfig
+
+
+def callback(config, level):
+    rospy.loginfo("""Server: task change request: {Njord_tasks}""".format(**config))
```

Comment:
Remove loginfo printing or set it as debug condition.

Reply:
should maybe be warning, if this is useful information to get while the drone is operating?